autogpt/memory/local.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, cfg) -> None:
         self.filename = f"{cfg.memory_index}.json"
         if os.path.exists(self.filename):
             try:
-                with open(self.filename, "w+b") as f:
+                with open(self.filename, "a+b") as f:
                     file_content = f.read()
                     if not file_content.strip():
```

Comment:
```suggestion
                    if not file_content.strip() and file_content.strip() != b'{}':
```
If `a+b` is the way to go (I need to sleep, was playing too much with this awesome tool) then this needs to happen as well or it will append `{}` each time.

Reply:
Got to the [pre-seeding portion](https://github.com/Significant-Gravitas/Auto-GPT#-memory-pre-seeding), perhaps this was intentional? Not sure if it is talking about the same thing.

> ⚠️For other memory backend, we currently forcefully wipe the memory when starting Auto-GPT. To ingest data with those memory backend, you can call the `data_ingestion.py` script anytime during an Auto-GPT run. 

If so, I can close this PR 😸